src/Core/Translation/Provider/CatalogueProviderFactory.php

```diff
@@ -57,43 +98,109 @@ class CatalogueProviderFactory
      */
     public function __construct(
         DatabaseTranslationLoader $databaseTranslationLoader,
```

Comment:
you add a lot of dependencies but the service definition doesn't seem to have changed, see `prestashop.translation.provider.catalogue_provider_factory` in `src/PrestaShopBundle/Resources/config/services/bundle/translation.yml`

Maybe I'm missing something?